setup.py

```diff
@@ -604,6 +604,17 @@ def run(self):
             # files
             ("%s/grub_config/grub" % libpath, glob("config/grub/grub/*")),
             # dirs
+            ("%s/boot" % tftproot, []),
+            ("%s/etc" % tftproot, []),
+            ("%s/grub" % tftproot, []),
+            ("%s/images" % tftproot, []),
+            ("%s/images2" % tftproot, []),
+            ("%s/ppc" % tftproot, []),
+            ("%s/ppc" % tftproot, []),
+            ("%s/s390x" % tftproot, []),
+            ("%s/pxelinux.cfg" % tftproot, []),
+            ("%s/system" % tftproot, []),
```

Comment:
Just to be sure.
So the dirs `system` and `system_link` are moved up from dir. `/var/lib/tftpboot/grub/` into new, separate directories `/var/lib/tftpboot/system` and `/var/lib/tftpboot/system_link`?

Take these files into consideration:
From cobbler-2.2.1-1-3477-g7d45ed59

```
# ag system_link
config/grub/README.grubconfig
51:grub/grub/system_link.*

cobbler/tftpgen.py
280:                        link_path = os.path.join(self.bootloc, "grub", "system_link", system.name)
```

Reply:
> Just to be sure.
> So the dirs `system` and `system_link` are moved up from dir. `/var/lib/tftpboot/grub/` into new, separate directories `/var/lib/tftpboot/system` and `/var/lib/tftpboot/system_link`?

Argh. No, this is a bug/regression. Thanks a lot for the review/heads-up!
I'll send a fix.